addons/mail/static/src/components/composer_text_input/composer_text_input.scss

```diff
@@ -20,7 +20,11 @@
     resize: none;
     border-radius: $o-mail-rounded-rectangle-border-radius-lg;
     border: none;
-    overflow: auto;
+    overflow: hidden;
+
+    &:placeholder-shown {
+        white-space: nowrap;
```

Comment:
Without ellipsis this looks bugged, but css ellipsis doesn't seem to have effect on placeholder?
Maybe we'll still need to do a js ellipsis on the placeholder value?

Reply:
Well I think it's more because the placeholder is on an textarea. You find code online that suggest that ellipsis works on input text.